src/app/personal-details/page.tsx

```diff
@@ -76,146 +60,9 @@ const PersonalDetailsScreen: React.FC = () => {
 
-				{/* Avatar Upload */}
-				{/* <Box
-          sx={{
-            display: "flex",
-            justifyContent: "center",
-            alignItems: "center",
-            gap: "32px",
-            marginBottom: "32px",
-          }}
-        >
-          <IconButton
-            sx={{
-              width: "80px",
-              height: "80px",
-              borderRadius: "50%",
-              backgroundColor: "#F0F0F5",
-              border: "2px dashed #D6D6E7",
-              "&:hover": {
-                backgroundColor: "#E8E8F0",
-              },
-            }}
-          >
-            <AddPhotoAlternateIcon
-              sx={{ fontSize: "40px", color: "#6C6C80" }}
-            />
-          </IconButton>
-          <Button
-            variant="outlined"
-            sx={{
-              textTransform: "none",
-              padding: "12px 36px",
-              fontSize: "16px",
-              borderColor: "#D6D6E7",
-              color: "#6C6C80",
-              "&:hover": {
-                borderColor: "#C0C0D8",
-              },
-              fontFamily: "'Outfit', sans-serif",
-              fontWeight: 400,
-              lineHeight: "20.16px",
-            }}
-          >
-            Upload Photo or Avatar
-          </Button>
-        </Box> */}
-
-				{/* Form Fields */}
-				<Box
-					sx={{
-						display: "flex",
-						flexDirection: "column",
-						gap: "40px",
-						marginBottom: "32px",
-						padding: "8px",
-					}}
-				>
-					{/* Name Field */}
-					<TextField
-						fullWidth
-						label="Name*"
-						value={name}
-						onChange={(e) => setName(e.target.value)}
-						variant="outlined"
-					/>
-
-					{/* Email Field */}
-					<TextField
-						fullWidth
-						label="Email*"
-						value={email}
-						onChange={(e) => setEmail(e.target.value)}
-						variant="outlined"
-					/>
-
-					{/* Age Field */}
-					<TextField
-						fullWidth
-						label="Age"
-						value={age}
-						onChange={(e) => setAge(e.target.value)}
-						variant="outlined"
-					/>
-
-					{/* Gender Field */}
-					<TextField
-						fullWidth
-						label="Gender"
-						value={gender}
-						onChange={(e) => setGender(e.target.value)}
-						variant="outlined"
-					/>
-
-					{/* Address Field */}
-					<TextField
-						fullWidth
-						label="Address"
-						value={address}
-						onChange={(e) => setAddress(e.target.value)}
-						variant="outlined"
-					/>
-
-					{/* Description Field */}
-					<TextField
-						fullWidth
-						label="What best describes you?"
-						value={description}
-						onChange={(e) => setDescription(e.target.value)}
-						variant="outlined"
-					/>
-
-					{/* Workplace Field */}
-					<TextField
-						fullWidth
-						label="Where do you work?"
-						value={workplace}
-						onChange={(e) => setWorkplace(e.target.value)}
-						variant="outlined"
-					/>
-
-					{/* LinkedIn URL Field */}
-					<TextField
-						fullWidth
-						label="LinkedIn URL*"
-						value={linkedin}
```

Comment:
I don't believe that the asterisk here is necessary.

Reply:
Good catch! Makes the changes. Thanks